geo/src/main/java/org/odk/collect/geo/selection/SelectionSummarySheet.kt

```diff
@@ -31,14 +33,32 @@ internal class SelectionSummarySheet(context: Context, attrs: AttributeSet?) :
 
         binding.name.text = item.name
 
-        binding.statusIcon.setImageDrawable(ContextCompat.getDrawable(context, item.status.icon))
-        binding.statusIcon.background = null
-        binding.statusText.text = item.status.text
+        binding.properties.removeAllViews()
+        item.properties.forEach {
+            val property = PropertyBinding.bind(
+                LayoutInflater.from(context).inflate(R.layout.property, binding.properties, false)
+            )
+
+            property.text.text = it.text
+
+            if (it.icon != null) {
+                property.icon.setImageDrawable(ContextCompat.getDrawable(context, it.icon))
+                property.icon.background = null
+            } else {
+                property.icon.visibility = View.GONE
+            }
+
+            binding.properties.addView(property.root)
```

Comment:
Shouldn't it be a list (recyclerview) instead of building it manually like that using `addView()`?

Reply:
Right now we're not optimizing for longer sets of properties (although we're interested to see if people end up with those) and we don't want the properties to be independently scrollable within the bottom sheet so a `LinearLayout` seems fine to me.